web/e2e-tests/navigation.test.ts

```diff
@@ -6,8 +6,36 @@ import * as common from "./lib/common";
 
 async function navigate_using_left_sidebar(page: Page, stream_name: string): Promise<void> {
     console.log("Visiting #" + stream_name);
-    await page.click(`.stream-name[title="${stream_name}"]`);
-    await page.waitForSelector(`#message_feed_container`, {visible: true});
+    await page.waitForSelector(".subscription_block", {visible: true, timeout: 20000});
```

Comment:
Hey @sayamsamal, This changes was done because the test was unexpectedly failing with the error log that the stream-name Verona is not found. That's why I used the wait for select for the class . subscription_block to wait so that it can find the stream name. If there is an alternative for this I would definitely follow that.

Reply:
Please take a look at https://github.com/zulip/zulip/pull/31851#issuecomment-2426579117.